crates/primitives/src/constants.rs

```diff
@@ -31,4 +31,7 @@ pub const TEST_PRIVATE_KEY: &str =
 pub const MIN_BASE_FEE_PER_GAS: u128 = 10_000_000; // 0.01 gwei
 
 /// Maximum size of a bitcoin transaction body in bytes
+#[cfg(not(feature = "test-utils"))]
```

Comment:
Although this works it introduces additional feature flag. We already have prover rpc for this to force it to prove, no?

Reply:
We do have rpc for forcing to prove. But we don't have rpc to force sequencer to submit a commitment to DA. I don't like too much feat flags either, but `test-utils` is used across our repository widely and might be the one that's actually useful.

Reply:
How about adding an rpc method?

Reply:
It might not be so useful tbh.

In all other tests we just keep min_seq_commitments config very low and easily force commitment. In this specific test, the aim is to force a seq comm by making it go above a size threshold. So we don't even test the commitment, we just test that the threshold triggers the action.

Reply:
why not go SHORT_PREFIX way?

Reply:
I mean instead of adding a feature

Reply:
makes more sense. done.